Location fix

marcvaneijk opened the pull request:
Changed resource location from parameter to resourceGroup().location.
Deleted location parameter from deployment template, nested templates
and parameter file.

singhkay commented:
@marcvaneijk  error ` AssertionError: spark-on-ubuntu/azuredeploy.parameters.json -  Parameter "storageAccountName" is missing its value field`

marcvaneijk commented:
@singhkay 
- moved storageAccountName to var
- replaced hardcoded endpoint with dynamic
- deleted storageAccountName param from param file
- Replaced fixed password with gen in param file
